### DISCUSSION TITLE
Shouldn't virtualizedfx be a module?

### DISCUSSION THREAD

**Discussion by kromar777** [palexdev/MaterialFX#144] [Archived]
I managed to get a simple hello world app to run with the latest materialfx with the help of extra-java-module-info gradle plugin (luckily i usually use it in my projects), otherwise i'm getting "error: module not found: virtualizedfx 
requires virtualizedfx;"
I didn't applied javafx plugin though as i use liberica jdk with bundled javafx sdk, maybe it would have worked.
Thus i needed all these lines in build.gradle to transform virtualizedfx to a module:
    module("kotlin-stdlib-common-1.5.21.jar", "kotlin", "1.5.21")
    automaticModule("virtualizedfx-11.2.4.jar", "virtualizedfx")
    module("gradle-maven-publish-plugin-0.18.0.jar", "gradle.maven.publish", "0.18.0")
    module("moshi-1.12.0.jar", "moshi", "1.12.0")

**Comment by palexdev** [comment]
Hello
`requires virtualizedfx;` should work as that's the same thing [MaterialFX](https://github.com/palexdev/MaterialFX/blob/main/materialfx/src/main/java/module-info.java) does.
  
If I remember correctly I avoided making it a module for two reasons:
1) I don't get why making a project which would have just one module, modular
2) I think I had some issues running tests or packaging the project, can't remember
